virttest/utils_conn.py

```diff
@@ -902,7 +902,7 @@ class TLSConnection(ConnectionBase):
                  'credential_dict', 'qemu_tls', 'qemu_chardev_tls',
                  'server_saslconf', 'server_qemuconf', 'client_qemuconf',
                  'server_libvirtd_tls_socket', 'client_libvirtd_tls_socket',
-                 'special_cn', 'server_setup_local',
+                 'special_cn', 'server_setup_local', 'server_info_ip',
```

Comment:
you may need add description after line 893 for your newly added attribute

Reply:
Yes, updated.